src/bp/core/mapping/mapping-table.ts

```diff
@@ -8,11 +8,11 @@ export class MappingTable extends Table {
 
     await this.knex.createTableIfNotExists(this.name, table => {
       table.string('scope')
+      table.uuid('local')
       table.string('foreign')
-      table.string('local')
-      table.primary(['scope', 'foreign', 'local'])
-      table.unique(['scope', 'foreign'])
+      table.primary(['scope', 'local', 'foreign'])
       table.unique(['scope', 'local'])
+      table.unique(['scope', 'foreign'])
```

Comment:
I think you should only change the local column type to uuid and left the rest untouched!

Reply:
I was kind of annoyed that sometimes in the code you had (local, foreing) and sometimes (foreign, local) in functions so I changed it to always be (local, foreign). On the db side I woulnd't have bothered changing it if the table was already in use, but since it's an unused table I changed the order here as well.